Formula/mg.rb

```diff
@@ -1,10 +1,8 @@
 class Mg < Formula
-  desc "Small Emacs-like editor"
-  # https://devio.us/~bcallah/mg/ is temporarily offline
+  desc "Small Emacs-like editor from OpenBSD"
```

Comment:
I don't really think the source is important

Reply:
Done